modules/backend/formwidgets/recordfinder/partials/_container.htm

```diff
@@ -2,10 +2,29 @@
 
     <span class="form-control"><?= e(trans('backend::lang.form.preview_no_record_message')) ?></span>
 
+<?php elseif ($field->readOnly): ?>
```

Comment:
Why can't this one be the same as the switch field? A much cleaner fix would be to simply add `|| $this->formField->readOnly` to https://github.com/octobercms/october/blob/master/modules/backend/formwidgets/RecordFinder.php#L179

Reply:
I'll test if that option passes the model validation.

Reply:
@LukeTowers it works but it didn't add the correct disabled color to the form field:
![image](https://user-images.githubusercontent.com/2809147/31512886-d8bf3582-af62-11e7-9af1-0a9f62aa2346.png)

Reply:
Add the disabled attribute to the preview mode markup then

Reply:
I've added the disabled attrute but it seems that the css its been overrided by another class as shown in the image below:

![image](https://user-images.githubusercontent.com/2809147/31513278-472ce5ae-af64-11e7-9e36-e50b1b1326bc.png)

>I don't want to toch the css files so I'll see if there is another class that I could use.

Reply:
The background-color of the disabled field should be **!important** I guess, what do you think ?

Reply:
How did it work with your code then?

Reply:
It work with my previous code because the form-control wasn't inside a div with the .field-recordfinder class. As shown in the picture below:

![image](https://user-images.githubusercontent.com/2809147/31514071-de5fde5c-af66-11e7-82e5-351d45b9e02b.png)

In the picture above the **<div class"form-control">** is inside a form group directly.

>Note this result is based on the first modification I made.

Reply:
Try changing the disabled CSS to !important then: https://github.com/octobercms/october/blob/develop/modules/system/assets/ui/less/form.base.less#L143. Build the assets locally with `php artisan october:util compile less` but don't commit the built CSS file, only your changes to the form.base.less file.

Reply:
It worked now! 👍  :)

![image](https://user-images.githubusercontent.com/2809147/31515306-14e8c58e-af6b-11e7-8337-d911fdb96edf.png)

I'll prepare the commit